src/api/handlers/webhook.ts

```diff
@@ -227,23 +226,29 @@ export function getSlashCommands(guildID?: string) {
 /**
  * Edit an existing slash command. If this command did not exist, it will create it.
  */
-export function upsertSlashCommand(options: UpsertSlashCommandOptions) {
+export function upsertSlashCommand(
+  commandID: string,
```

Comment:
What is a commandID?

Reply:
Its the ID od the command.
https://discord.com/developers/docs/interactions/slash-commands#edit-global-application-command

Reply:
What if the command does not exist?

Reply:
Right we should check that before sending the request.

Reply:
But the parameter `commandID` is not optional, but previously it was.

Reply:
No `guildID` was optional

Reply:
No? you need it for the URL

Reply:
I still don't know what a commandID is

Reply:
Every command has a unique ID. You need to pass this ID so discord knows what command you want to edit.

![image](https://user-images.githubusercontent.com/72305210/104024807-98d8a080-51bb-11eb-8613-7c41d4636e6e.png)